db/db_impl_compaction_flush.cc

```diff
@@ -1491,7 +1492,9 @@ void DBImpl::BackgroundCallCompaction(PrepickedCompaction* prepicked_compaction,
       // It also applies to access other states that DB owns.
       log_buffer.FlushBufferToLog();
       if (job_context.HaveSomethingToDelete()) {
+        TEST_SYNC_POINT("DBImpl::BackgroundCallCompaction:3");
         PurgeObsoleteFiles(job_context);
+        TEST_SYNC_POINT("DBImpl::BackgroundCallCompaction:3");
```

Comment:
the sync point with the same name above is probably unneeded.

Reply:
Done.